metering/configuring_metering/metering-common-config-options.adoc

```diff
@@ -82,22 +82,26 @@ spec:
   reporting-operator:
     spec:
       nodeSelector:
-        "node-role.kubernetes.io/infra": "true"
+        "node-role.kubernetes.io/infra": "" <1>
 
   presto:
     spec:
       coordinator:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
       worker:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
   hive:
     spec:
       metastore:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
       server:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
 ----
+<1> Add a `nodeSelector` parameter with the appropriate value to the component you want to move. You can use a `nodeSelector` in the format shown or use key-value pairs, based on the value specified for the node.
```

Comment:
I think we may want to convey the need for verifying that the labels on these nodes exist too.

Reply:
@lbarbeevargas @timflannagan1 I think putting an example value here is beneficial to the end-users.

Reply:
@timflannagan1  @pruan-rht Can you please provide an example value for this callout or value(s) I can add back in to the yaml example? Thank you!

Reply:
> I think we may want to convey the need for verifying that the labels on these nodes exist too.

@timflannagan1 Do you want me to include this verification? Please provide the steps and example code if yes. Thank you!

Reply:
I think the present, `"node-role.kubernetes.io/infra": ""` node selector configuration is sufficient, despite infra nodes not being as prevalent in 4.x. @pruan-rht Is there an example you had in mind?

Reply:
@timflannagan1 if `""` works then it's okay.